docs/reference/index-modules/store.asciidoc

```diff
@@ -16,7 +20,7 @@ This can be overridden for all indices by adding this to the
 
 [source,yaml]
 ---------------------------------
-index.store.type: niofs
+index.store.type: mmapfs
```

Comment:
should this instead be `fs` (i.e. the default, which is hybridfs)?

Reply:
The above line says that "[Defaults] can be overridden" this way, so I thought that it might be confusing to use the default store type in this example?

Reply:
Perhaps we can change the phrasing to:

> The storage type can also be explicitly set for all indices by configuring the store type in the `config/elasticsearch.yml` file:

and then use `hybridfs` as an example?